src/main/resources/data/socratesAtlasConfiguration.yaml

```diff
@@ -0,0 +1,6 @@
+hikariConfigs:
```

Comment:
Can you please put that into a different folder for example `src/main/resources/config` ? Because this is not a data file, but a config for connection